src/main/java/appeng/client/gui/IInterfaceTerminalPostUpdate.java

```diff
@@ -0,0 +1,16 @@
+package appeng.client.gui;
+
+import java.util.List;
+
+import appeng.core.sync.packets.PacketIfaceTermUpdate.PacketEntry;
+
+public interface IInterfaceTerminalPostUpdate {
+
+    /**
+     * Interface to handle updates inside interface terminal
+     *
+     * @param updates     List of updates
+     * @param statusFlags status bitflags
+     */
+    void postUpdate(List<PacketEntry> updates, int statusFlags);
```

Comment:
Not so sure about creating a whole new interface for this. Does ae2fc not inherit off GuiInterfaceTerminal already to achieve the same effect

Reply:
Yes it implement same interface and packet can be delivered without any dirty hacks